--- zstd-1.5.2/build/meson/lib/meson.build~	2022-01-20 21:17:18.000000000 +0000
+++ zstd-1.5.2/build/meson/lib/meson.build	2022-06-20 16:05:17.931504782 +0000
@@ -112,6 +112,7 @@
   endif
 endif
 libzstd_c_args += cc.get_supported_arguments(libzstd_debug_cflags)
+libzstd_c_args += '-std=gnu11'
 
 libzstd = library('zstd',
   libzstd_sources,
--- zstd-1.5.2/build/meson/programs/meson.build~	2022-01-20 21:17:18.000000000 +0000
+++ zstd-1.5.2/build/meson/programs/meson.build	2022-06-20 16:13:44.177689821 +0000
@@ -26,6 +26,7 @@
 if use_multi_thread
   zstd_c_args += [ '-DZSTD_MULTITHREAD' ]
 endif
+zstd_c_args += '-std=gnu11'
 
 zstd_deps = [ libzstd_dep ]
 if use_zlib
